core/db/migrate/20200607161221_add_store_owner_order_notification_delivered_to_spree_orders.rb

```diff
@@ -0,0 +1,5 @@
+class AddStoreOwnerOrderNotificationDeliveredToSpreeOrders < ActiveRecord::Migration[6.0]
+  def change
+    add_column :spree_orders, :store_owner_notification_delivered, :boolean, default: false
```

Comment:
@imderek let's avoid setting the default value here - for large date sets this will be extremely slow